### DISCUSSION TITLE
Form placed inside a <template> tag on a livewire component doesn't register any of the livewire events

### DISCUSSION THREAD

**Discussion by abishekrsrikaanth** [livewire/livewire#4535] [1. Help]
I have a laravel livewire component with a form using `wire:submit.prevent`.

The form itself is inside a `<template>` tag that shows the content based on an alpine `x-if` condition.

The form submission does not work if the livewire form is inside a `template` tag. 

Example: https://laravelplayground.com/#/snippets/ecbf5018-52f3-4f1c-a9f9-121c878467a6

@joshhanley @calebporzio Any idea if this is a known issue or a bug. is there a good way to fix this? Got an app for production release and can't push till there is a way to fix this. Appreciate your time in looking into this.

Currently using
Alpine:  v3.4.2
Laravel Livewire: v2.9.0
Laravel: v8.79.0
Browser: All browsers

**Comment by joshhanley** [comment]
@abishekrsrikaanth Livewire won't work inside template tags as Livewire needs the elements to exist on the page, so it can initialise everything properly.

So you will need to change it to use `x-show` and remove the template tag, or if you don't want it in the DOM, then make use of blade `@if` to hide the form, but that will require a request to the server to show the form.

Hope this helps!

**Reply by abishekrsrikaanth** [reply]
Thanks @joshhanley . Is this documented on the docs? Would be great if this is documented somewhere

**Reply by joshhanley** [reply]
@abishekrsrikaanth I don't think it is documented. But I'm not sure where the best place for that would be. PR's to the docs are welcome 🙂 https://github.com/livewire/docs
